File: zwkfb_view/src/main/java/安卓/组件/图像视图.java
package 安卓.组件;

import android.annotation.SuppressLint;
import android.content.Context;
import android.util.AttributeSet;
import android.widget.ImageView;

import androidx.annotation.Nullable;

/**
 * 创建时间：2025年11月18日.
 * <p>
 * 描述：图像视图
 * @author dxyc
 */
@SuppressLint("AppCompatCustomView")
public class 图像视图 extends ImageView {

    public 图像视图(Context 上下文) {
        super(上下文);
    }

    public 图像视图(Context 上下文, @Nullable AttributeSet 属性) {
        super(上下文, 属性);
    }

    public 图像视图(Context 上下文, @Nullable AttributeSet 属性, int 默认样式属性) {
        super(上下文, 属性, 默认样式属性);
    }

    public 图像视图(Context 上下文, @Nullable AttributeSet 属性, int 默认样式属性, int 默认样式资源) {
        super(上下文, 属性, 默认样式属性, 默认样式资源);
    }


}

File: zwkfb_view/src/main/java/安卓/组件/图像视图.kt
package 安卓.组件

import android.annotation.SuppressLint
import android.content.Context
import android.graphics.ColorFilter
import android.graphics.PorterDuff
import android.util.AttributeSet
import android.widget.ImageView


//======================================================================

/**
 * 描述：颜色过滤器
 *
 * 版本：0.1.1
 * @param 颜色过滤器
 * @return 颜色过滤器
 */
var ImageView.颜色过滤器: ColorFilter
    get() = colorFilter
    set(颜色过滤器) {
        colorFilter = 颜色过滤器
    }
/**
 * 描述：取颜色过滤器
 *
 * 版本：0.1.1
 * @return 颜色过滤器
 */
fun ImageView.取颜色过滤器(): ColorFilter {
    return getColorFilter()
}
/**
 * 描述：置颜色过滤器
 *
 * 版本：0.1.1
 * @param 颜色过滤器
 */
fun ImageView.置颜色过滤器(颜色过滤器: ColorFilter) {
    setColorFilter(颜色过滤器)
}

//======================================================================
/**
 * 描述：置颜色过滤器
 *
 * 版本：0.1.1
 * @param 颜色 颜色
 */
fun ImageView.置颜色过滤器(颜色: Int) {
    setColorFilter(颜色)
}

/**
 * 描述：置颜色过滤器
 *
 * 版本：0.1.1
 * @param 颜色 颜色
 * @param 模式 模式
 */
fun ImageView.置颜色过滤器(颜色: Int, 模式: PorterDuff.Mode) {
    setColorFilter(颜色, 模式)
}

//======================================================================


